Resources/Textures/Nyanotrasen/Objects/Weapons/Melee/iceaxe.rsi/meta.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "katayusha",
+    "size": {
+      "x": 32,
+      "y": 32
+    },
+    "states": [
+      {
+        "name": "icon"
+      }
```

Comment:
Could you do inhand sprites for this? Weapons are one of the important things that should be visible when someone's holding them.

Reply:
i can try, i might not get around to it until later. i want to get the pebble update and the plushie out first .